contrib/test/ci/build/cri-o.yml

```diff
@@ -8,12 +8,12 @@
 
 - name: check if cri-o.tar.gz exists
   stat:
-    path: /root/cri-o.tar.gz
+    path: "/home/deadbeef/cri-o.tar.gz"
```

Comment:
How about `/home/foobar`?

Reply:
I think it would be kinda weird to put it in a place that's not the homedir of the user

Reply:
ah, my bad